go-sdk/generate.sh

```diff
@@ -39,8 +39,8 @@ cp $SCRIPT_DIR/../common/src/main/resources/META-INF/openapi-v2.json $SCRIPT_DIR
 cp $SCRIPT_DIR/../common/src/main/resources/META-INF/openapi.json $SCRIPT_DIR/v3.json
 
 # Hask to overcome https://github.com/microsoft/kiota/issues/3920
-$SED_NAME -i 's/NewComment/DTONewComment/' v2.json
-$SED_NAME -i 's/NewComment/DTONewComment/' v3.json
+$SED_NAME -i 's/NewComment/DTONewComment/' $SCRIPT_DIR/v2.json
```

Comment:
Good one!
I think that this has been fixed in upstream Kiota, it might be time to remove this hack

Reply:
I'm still getting
```
# github.com/apicurio/apicurio-registry/go-sdk/pkg/registryclient-v3/models
pkg/registryclient-v3/models/new_comment.go:8:6: NewComment redeclared in this block
	pkg/registryclient-v3/models/comment.go:23:6: other declaration of NewComment
```
when running `make test`.

Reply:
ok, let's test the fix next time 👍